research-software-directory/RSD-as-a-service
# Upgrade react testing library from v14 to v16

# Upgrade unit testing library (frontend)

Closes #1275

Changes proposed in this pull request:
* Upgrade react testing library from v14 to v16
* Fix broken test for AddPageModal component      

How to test:
* navigate to frontend folder.
* remove node_modules `rm -rf node_modules`
* install dependencies `yarn install`
* run unit tests `yarn test`
* run build `yarn build`
* confirm that tests on this PR passing
* build app `make start`
* navigate through app and confirm that it works properly

PR Checklist:
